tests/acceptance/features/webUIFiles/copyCurrentPath.feature

```diff
@@ -0,0 +1,13 @@
+Feature: copy current path as a permanent link
+  As a user
+  I want to copy the permanent link to the current folder
+  So that I can paste paste it else where
+
+  Background:
+    Given user "user1" has been created with default attributes
+    And user "user1" has logged in using the webUI
+    And the user has browsed to the files page
+
+  Scenario: Copy permalink to clipboard
+    When the user copies permalink to the current folder
+    Then the clipboard content should match current folder permalink
```

Comment:
what is the current folder? maybe better just to pass in the folder name, in the current case "/"

Reply:
sure, but I need to get it working at least with a root dir first

Reply:
done. 'current' is a folder  that is opened currently in browser. 
And UI element provides a permanent link exactly to the currently opened folder